common/src/dodeca.rs

```diff
@@ -171,234 +169,305 @@ impl Vertex {
     pub fn dual_to_node(self) -> &'static na::Matrix4<f32> {
-        &DUAL_TO_NODE_F32[self as usize]
+        &dual_to_node_f32()[self as usize]
     }
 
     /// Transform from cube-centric coordinates to dodeca-centric coordinates
     pub fn dual_to_node_f64(self) -> &'static na::Matrix4<f64> {
-        &DUAL_TO_NODE_F64[self as usize]
+        &dual_to_node_f64()[self as usize]
     }
 
     /// Transform from dodeca-centric coordinates to cube-centric coordinates
     pub fn node_to_dual(self) -> &'static na::Matrix4<f32> {
-        &NODE_TO_DUAL_F32[self as usize]
+        &node_to_dual_f32()[self as usize]
     }
 
     /// Transform from dodeca-centric coordinates to cube-centric coordinates
     pub fn node_to_dual_f64(self) -> &'static na::Matrix4<f64> {
-        &NODE_TO_DUAL_F64[self as usize]
+        &node_to_dual_f64()[self as usize]
     }
 
     /// Scale factor used in conversion from cube-centric coordinates to euclidean chunk coordinates.
     /// Scaling the x, y, and z components of a vector in cube-centric coordinates by this value
     /// and dividing them by the w coordinate will yield euclidean chunk coordinates.
     pub fn dual_to_chunk_factor() -> f32 {
-        *DUAL_TO_CHUNK_FACTOR_F32
+        dual_to_chunk_factor_f32()
     }
 
     /// Scale factor used in conversion from cube-centric coordinates to euclidean chunk coordinates.
     /// Scaling the x, y, and z components of a vector in cube-centric coordinates by this value
     /// and dividing them by the w coordinate will yield euclidean chunk coordinates.
     pub fn dual_to_chunk_factor_f64() -> f64 {
-        *DUAL_TO_CHUNK_FACTOR_F64
+        dual_to_chunk_factor_f64()
     }
 
     /// Scale factor used in conversion from euclidean chunk coordinates to cube-centric coordinates.
     /// Scaling the x, y, and z components of a vector in homogeneous euclidean chunk coordinates by this value
     /// and lorentz-normalizing the result will yield cube-centric coordinates.
     pub fn chunk_to_dual_factor() -> f32 {
-        *CHUNK_TO_DUAL_FACTOR_F32
+        chunk_to_dual_factor_f32()
     }
 
     /// Scale factor used in conversion from euclidean chunk coordinates to cube-centric coordinates.
     /// Scaling the x, y, and z components of a vector in homogeneous euclidean chunk coordinates by this value
     /// and lorentz-normalizing the result will yield cube-centric coordinates.
     pub fn chunk_to_dual_factor_f64() -> f64 {
-        *CHUNK_TO_DUAL_FACTOR_F64
+        chunk_to_dual_factor_f64()
     }
 
     /// Convenience method for `self.chunk_to_node().determinant() < 0`.
     pub fn parity(self) -> bool {
-        CHUNK_TO_NODE_PARITY[self as usize]
+        chunk_to_node_parity()[self as usize]
     }
 }
 
-pub const VERTEX_COUNT: usize = 20;
-pub const SIDE_COUNT: usize = 12;
-pub const BOUNDING_SPHERE_RADIUS_F64: f64 = 1.2264568712514068;
-pub const BOUNDING_SPHERE_RADIUS: f32 = BOUNDING_SPHERE_RADIUS_F64 as f32;
+pub mod gridding {
```

Comment:
I wonder what this module should be called. I expect that `gridding` would be confusing because `dodeca` doesn't care about individual voxels. Some names that pop into my head are `internal_data`, `lazy_static`, `lazy_data`, or `once_locks`, but none of them feel that great.

If I had to decide, I would just call it `internal_data`, but I'm wondering if you or @Ralith have any thoughts here.

Reply:
That'd be fine, or `data`, `tables`, `details`, anything like that.